hw8/createdApp/src/components/dayWeather.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import WeatherService from '../services/weatherService';
```

Comment:
По импортам, для лучшей читаемости, можно разделять их на 2 категории, где сначала идут импорты из сторонних библиотек, а потом уже свои файлы
```suggestion
import React from 'react';
import moment from 'moment';

import WeatherService from '../services/weatherService';
```
Еще заметил что не везде стоят точки с запятыми, либо убери их везде, либо ставь их везде, чтобы было легче читать) Штуки типа ESLint или Prettier могут помочь это автоматизировать